src/style.css

```diff
@@ -0,0 +1,101 @@
+/* #region fonts */
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: heavy;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: book;
+  font-weight: normal;
+}
+/* #endregion */
+*:focus {
+  outline: none;
+}
```

Comment:
`input:focus`

Don't increase specific

Reply:
Sorry. Thought that we should turn of outline focus. Removed.